app/Models/User.php

```diff
@@ -29,7 +29,7 @@ class User extends SnipeModel implements AuthenticatableContract, AuthorizableCo
     use CompanyableTrait;
 
     protected $presenter = \App\Presenters\UserPresenter::class;
-    use SoftDeletes, ValidatingTrait;
+    use SoftDeletes, ValidatingTrait, Loggable;
```

Comment:
This is probably fine, but seems a little out-of-nowhere?

Reply:
It wasn't logging the upload without it